recipes/ftd2xx/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ftd2xx" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 90df60c2468e7dd9c870fcfe6dff8da6b477334f912e5028c3ccf54a78466f3a
+  patches:
+    - 20-avoid_unrecoverable_errors.patch  # [not py27]
```

Comment:
Is this patch purely for the test? If so you could catch `SystemExit` instead of `OSError`.

Reply:
No, this patch is because because I don't want my users python to just completely die when the dll isn't there. 

I don't really care about python 2 and so I don't have the bandwidth to think of how to make it compatible with both. I submitted this upstream

Reply:
As a note, it is only applicable to windows. Other OSes go through a different path.